packages/ionic/src/layouts/categorization/categorization-menu-layout.ts

```diff
@@ -51,7 +51,7 @@ import { JsonFormsIonicLayout } from '../JsonFormsIonicLayout';
       </ion-content>
   `
 })
-export class CategorizationLayoutRenderer extends JsonFormsIonicLayout {
+export class CategorizationMenuLayoutRenderer extends JsonFormsIonicLayout {
```

Comment:
Is this renderer event needed ? the tab based one will always win ...

Reply:
I think we should keep it for now, but the default should be the tab one.